Assets/Global/Scripts/Enemies/EnemyBase.cs

```diff
@@ -1,9 +1,9 @@
 using UnityEngine;
 
 // Base enemy class
-public class EnemyScript : MonoBehaviour
+public abstract class EnemyBase : MonoBehaviour
```

Comment:
base classes should be abstracts.
Base basically means its an incomplete class that should not be instantiated yet. and thats exactly what abstract is used for